.github/workflows/ci.yml

```diff
@@ -1087,6 +1087,104 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  # Runs tests on `tests/cli/fluvio_components`
+  fluvio_components:
```

Comment:
should rename as partition test

Reply:
Sure now its called `partition test` as well as the Test Suite